core/transaction.go

```diff
@@ -376,7 +376,7 @@ func VerifyTransactions(txs []Transaction, n utils.Network, protocolVersion stri
 	return nil
 }
 
-const commitmentTrieHeight uint = 64
+const commitmentTrieHeight uint8 = 64
```

Comment:
Do we need to specify type here? In Go constants are untyped by default

Reply:
Removed the type